cmd/syncthing/main.go

```diff
@@ -987,6 +989,9 @@ func loadConfigAtStartup() (config.Wrapper, error) {
 	}
 
 	if cfg.RawCopy().OriginalVersion != config.CurrentVersion {
+		if cfg.RawCopy().OriginalVersion > config.CurrentVersion && !forceNewConfig {
+			return nil, errors.New("Config file version (" + strconv.Itoa(cfg.RawCopy().OriginalVersion) + ") is newer than supported version (" + strconv.Itoa(config.CurrentVersion) + "). If this is expected, use -force-new-config to override.")
```

Comment:
`fmt.Errorf("bla bla %d bla", number)`

Also, the flag is not forcing anything, but allowing, so perhaps -allow-newer-config